### ISSUE TITLE
UI Scaling broken in most recent version

### ISSUE THREAD

**Issue by Billology** [WeakAuras/WeakAuras2#1103]
**What steps will reproduce the problem?**
Loading my auras using the most recent version, 
looks like this: https://i.imgur.com/vw2swhS.png 
Instead of this: https://i.imgur.com/sLLyPci.png

**Did you try having WeakAuras as the only enabled addon and everything else (especially something like ElvUI) disabled?**
Yes.

**Which version of WeakAuras are you using?**
2.11.0 results in screenshot 1 

**Was it working in a previous version? If yes, which was the last good one?**
2.10.11 results in screenshot 2

**Do you have an error log of what happened?** 
No

**Please provide any additional information below:**
I'm using UIScale 0.7111 and 1920x1080 resolution. 

**In case you do know which aura is problematic, please export the string and paste it below.**

``` !DAvBVjooq4Fn3hPIqaQ6hUtQaHRi1nWHd79Iw5Il2j4To25CC6BAf)2VzStaA3BpvDsvbNjZ4zMhpppU0iAgLiPPJIh1F4ikPMsgCru)lIIOKskHJ)bEuxXSp87sUBpnnQp8UJT7HAI8vb8(akH142BSlRCsJUgcW15YCfRa2ZuJwqjVSmpVw4GqUecrVdczLrQbNNMKMLSMsU3y5c7uJYyHDjnIM2NMoW)m2)CiAecWACmNit8mSoDzAcSF76soMARlufsTSDvoSSEVFnAFNrF3UZYtKppr(8eDkp1cv(7Rr2BkqFGZhpkE444Hxc)eF1OH9P59gf)(IMlQzUglu407BX0aI8NOBeLH5b7MAXD7uS6A6ekjSaBkPUqbG4S1BwmdoCAuozx3uxj2H(8wJ4PZ7mcMDYsX33a)lDEPX42VYAkSIqPa1118V2u7eCb)tsn2ewrbG6zVujWHaldaNUJEmehCa1yHVLOECZIdBNOy6hcMNBqqDUv(6HT)wdJdWc7W2SSJto)4VJahMCFR8J9JPQ2ZWUH0iHbzDDsTU6V)Q9pzHk4dacH5Xjq9WTMkkzIs(6RmlhsGXaWkyAhaiMsCsCtfhpAjnT)I54efXzLffcBDiHzhn0EINYkfiIfA(LvcOpGQJ8l)myR5EXJcTJ0KNlFMsUB61KS7izxVgI18KEPw9cgR3jk5gbtbe1JHTYk8HrwLC7TqEpEyn44rfjYpePul4T0xnupTlBqoePsXEbQwFGDFeXwIrXSh2MuwzEsylXkW57iYXPaU4EOYddj3KC7Q5BUnSVNGaFqif(rrwW4NmCa56bQnEASty1m1NbaeM3GehHkiYsgs6pNZ32EAVGdhjBGd3fmwlawp0GWgwYK6pU3h1o(y(JdoTJml0ZT4a69HXbpV3j39WlZAd(OsWhyA8vJP0RuSxil27csXbsW5YSp3kZoV34444bdU6QbJgpEWLdqvPHJBZ3eLqZ9Gm56zZovX)31r)lgDIx0oC39AN6ExrHTgURfwtJM))uZeUzXGZqHuCAvcVaQ8OdBxjFwOOKNcxofpUZHoKFsqShlgVOFTeB51l(1B6KswJ3L4NQ63A5gjNl0aEM8zuXpyRfZJ73QFDoKFMw(Fe2g)W)h4m1JuNuj4YAKOHYjR95J8tGgdkF177jAh2(TVDyRNNDyB076M2J2nzlH5xlqybbhqA0FEfU7DoAnWkjtxNKKEc6c3s3UJzDI4lqwyoBN4ltzWuSUycZ(Lnl6D6TEeV2mWxFeyQ(R5E7TgSNrBgRek)aOtUz56f)1Y0SRbTjKbjp7FI4m44wroIgAVt5YIorJhHcD4aelP)Zp ```

**Comment by Billology** [comment]
I tested back and forth between the versions a few times, insuring the scaling was the same.

**Comment by InfusOnWoW** [comment]
No this is a bit different, than #236 

In 8.1, Blizzard changed how textures work to automatically align to a pixel grid. Which leads to various problems, e.g. progress textures animating very strangely. In 2.11 we disabled that, which fixed that bug.

Unfortunately Blizzard forgot to add the same api to backdrops/borders.

Now, there might be something more going on here as the user is apparently using the right scaling factor and thus the snapping to pixel grid shouldn't matter. I need to look at that in game.

**Comment by InfusOnWoW** [comment]
Sorry for taking a while to come back to this. The aura you posted unfourtanately uses a non standard border texture, can you run /dump LibStub("LibSharedMedia-3.0"):Fetch("border", "1 Pixel") and tell us what that outputs?

**Comment by InfusOnWoW** [comment]
It's "Interface\\Buttons\\WHITE8X8", so let me try again.

**Comment by InfusOnWoW** [comment]
This actually turned out to be unrelated to the change in 2.11, it was just surfaced in your setup by 2.11. I've now changed something for the next release which should help this case.
